app/assets/javascripts/fae/form/inputs/_text.js

```diff
@@ -56,6 +57,38 @@ Fae.form.text = {
     $('.js-markdown-editor:not(.mde-enabled)').each(function () {
       var $this = $(this);
 
+      // set up translate button for markdown fields and add helper text
+      var translate_button = $this.siblings( '.js-translate-button' )
+      if (translate_button.length) {
```

Comment:
the `length` property on the jquery object will still be present even if it doesn't find the element so this should check to ensure `length > 0`

Reply:
updated!